Makefile

```diff
@@ -5,20 +5,37 @@ BROKER_IMAGE     ?= $(REGISTRY)/$(ORG)/ansible-service-broker
 BUILD_DIR        = "${GOPATH}/src/github.com/openshift/ansible-service-broker/build"
 PREFIX           ?= /usr/local
 BROKER_CONFIG    ?= $(PWD)/etc/generated_local_development.yaml
+SOURCE_DIRS      = cmd pkg
 SOURCES          := $(shell find . -name '*.go' -not -path "*/vendor/*")
 SVC_ACCT_DIR     := /var/run/secrets/kubernetes.io/serviceaccount
 KUBERNETES_FILES := $(addprefix $(SVC_ACCT_DIR)/,ca.crt token tls.crt tls.key)
+LOCAL_CI         ?= true
 .DEFAULT_GOAL    := build
 
-vendor:
+vendor: ## Install project dependencies
 	@glide install -v
 
-broker: $(SOURCES)
-	go build -i -ldflags="-s -w" ./cmd/broker
+broker: $(SOURCES) ## Build the broker
+	@go build -i -ldflags="-s -w" ./cmd/broker
 
-build: broker
+build: vendor broker ## Build the project
```

Comment:
Don't add vendor here. We don't need to update vendor since (sadly) it is checked into the repo.